calibrators/marker_radar_lidar_calibrator/src/marker_radar_lidar_calibrator.cpp

```diff
@@ -1249,134 +1398,107 @@ std::pair<double, double> ExtrinsicReflectorBasedCalibrator::computeCalibrationE
     initial_distance_error, initial_yaw_error);
   RCLCPP_INFO(
     this->get_logger(),
     "Final calibration error: detection2detection.distance=%.4fm yaw=%.4f degrees",
-    calibrated_2d_distance_error, calibrated_2d_yaw_error);
-  RCLCPP_INFO(
-    this->get_logger(),
-    "Final calibration error (rotation only): detection2detection.distance=%.4fm yaw=%.4f degrees",
-    calibrated_rotation_distance_error, calibrated_rotation_yaw_error);
+    calibrated_distance_error, calibrated_yaw_error);
 
-  auto [calibrated_2d_translation_difference, calibrated_2d_rotation_difference] =
-    compute_transformation_difference(
-      initial_radar_to_lidar_eigen_, calibrated_2d_radar_to_lidar_transformation);
-  auto [calibrated_rotation_translation_difference, calibrated_rotation_rotation_difference] =
+  auto compute_transformation_difference =
+    [](const Eigen::Isometry3d & t1, const Eigen::Isometry3d & t2) -> std::pair<double, double> {
+    double translation_difference = (t2.inverse() * t1).translation().norm();
+    double rotation_difference =
+      std::acos(std::min(1.0, 0.5 * ((t2.rotation().inverse() * t1.rotation()).trace() - 1.0)));
+
+    return std::make_pair(translation_difference, rotation_difference);
+  };
+
+  auto [calibrated_translation_difference, calibrated_rotation_difference] =
     compute_transformation_difference(
-      initial_radar_to_lidar_eigen_, calibrated_rotation_radar_to_lidar_transformation);
+      initial_radar_to_lidar_eigen_, calibrated_radar_to_lidar_transformation);
 
   std::unique_lock<std::mutex> lock(mutex_);
   if (
-    calibrated_2d_translation_difference < parameters_.max_initial_calibration_translation_error &&
-    calibrated_2d_rotation_difference < parameters_.max_initial_calibration_rotation_error) {
+    calibrated_translation_difference < parameters_.max_initial_calibration_translation_error &&
+    calibrated_rotation_difference < parameters_.max_initial_calibration_rotation_error) {
     RCLCPP_INFO(
-      this->get_logger(), "The 2D calibration pose was chosen as the output calibration pose");
-    calibrated_radar_to_lidar_eigen_ = calibrated_2d_radar_to_lidar_transformation;
-    calibration_valid_ = true;
-    calibration_distance_score_ = calibrated_2d_distance_error;
-    calibration_yaw_score_ = calibrated_2d_yaw_error;
-  } else if (
-    calibrated_rotation_translation_difference <
-      parameters_.max_initial_calibration_translation_error &&
-    calibrated_rotation_rotation_difference < parameters_.max_initial_calibration_rotation_error) {
-    RCLCPP_WARN(
-      this->get_logger(),
-      "The pure rotation calibration pose was chosen as the output calibration pose. This may mean "
-      "you need to collect more points");
-    calibrated_radar_to_lidar_eigen_ = calibrated_rotation_radar_to_lidar_transformation;
+      this->get_logger(), "The calibration pose was chosen as the output calibration pose");
```

Comment:
The output messages in this method make sense in the context of the code before this PR. Now they do not make much sense. Please fix them

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/180/commits/9bee4d290ba74eea1a9ccc7c45a7a745f2954e78